tests/client-dbus/tests/dbus/filesystem/test_properties.py

```diff
@@ -78,9 +79,12 @@ def testProps(self):
         # I think this is also always true
         self.assertEqual(len(created), 20)
 
-        used = Filesystem.Properties.Used.Get(filesystem)
+        (used_success, used) = FetchProperties.Methods.GetProperties(
+            filesystem, {"properties": ["Used"]}
+        )["Used"]
 
-        self.assertEqual(used, "12345678")
+        self.assertEqual(used_success, True)
+        self.assertEqual(used, "12345678 bytes")
```

Comment:
I don't like that the result contains "bytes", we should be sending the bare number over the D-Bus. To fix this, you have to dereference the value, so it isn't Bytes but just a number and _then_ stringify it. We should make sure to do that consistently everywhere.

Reply:
In that case, I'd argue it should be an integer instead of a string. If we're not including units, I don't see a reason to encode it as a string.

Reply:
The reason it was a string was that the integers only go up to u64. With dedup and compression certain sizes that we report could be ridiculously large, larger than would fit in a 64 bit integer. So it is safer long-term to use the string.

Reply:
Resolved.